src/Uno.Templates/content/unoapp/GitHub/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
```

Comment:
Does this automatically enable dependabot, or does the developer still need to add dependabot? If the latter, lets remove thisd